src/index.tsx

```diff
@@ -10,6 +10,17 @@ import "./assets/styles/global.scss";
 async function renderApp() {
   try {
     await Arc.InitializeArcJs();
+    // Silence 240 sec error
+    Arc.ContractWrappers.AbsoluteVote.contract.constructor.synchronization_timeout = 0;
```

Comment:
I hope this is fixed by the work @dkent600 is doing to change how arc.js handles transactions

Reply:
@tibetsprague Without changing the timeout, truffle will still timeout waiting for mined transactions.